### PR TITLE
Update and rename Samsung_CN.xml to Samsung-cn.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

**Comment by ivysrono** [comment]
想统一下，仅在 Only available in China 时用 _CN

**Comment by ivysrono** [comment]
Only use _CN while "Only available in China".

File: src/chrome/content/rules/Samsung-cn.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	For other Samsung coverage, see Samsung.com.xml.
+
+	Check error:Non-2xx HTTP code
+		http://samsung.cn/ (200) => https://www.samsung.com/cn/ (403)
+		http://www.samsung.cn/ (200) => https://www.samsung.com/cn/ (403)
+		http://samsung.com.cn/ (200) => https://www.samsung.com/cn/ (403)
+		http://www.samsung.com.cn/ (200) => https://www.samsung.com.cn/ (403)
+
+	Problematic domains:
+		- (www.)samsung.cn	(→ www.samsung.com/cn)
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Samsung-cn.xml]: Mismatch

File: src/chrome/content/rules/Samsung-cn.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<!--
+	For other Samsung coverage, see Samsung.com.xml.
+
+	Check error:Non-2xx HTTP code
+		http://samsung.cn/ (200) => https://www.samsung.com/cn/ (403)
+		http://www.samsung.cn/ (200) => https://www.samsung.com/cn/ (403)
+		http://samsung.com.cn/ (200) => https://www.samsung.com/cn/ (403)
+		http://www.samsung.com.cn/ (200) => https://www.samsung.com.cn/ (403)
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Samsung-cn.xml]: Updates needed.

File: src/chrome/content/rules/Samsung-cn.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+		- (www.)samsung.cn	(→ www.samsung.com/cn)
+
+	Mismatch:
+		- content.samsung.cn
+-->
+
+<ruleset name="Samsung-cn">
+
+	<target host="res.samsungshop.com.cn" />
+	<target host="www.samsungshop.com.cn" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Samsung-cn.xml]: Add samsungshop.com.cn
Move `www`

**Comment by gloomy-ghost** [comment]
```
ERROR src/chrome/content/rules/Samsung-cn.xml: Non-2xx HTTP code: http://samsung.com.cn/ (200) => https://www.samsung.com.cn/ (403)
ERROR src/chrome/content/rules/Samsung-cn.xml: Fetch error: http://samsungshop.com.cn/ => https://samsungshop.com.cn/: (28, 'Operation timed out after 0 milliseconds with 0 out of 0 bytes received')
```

@Hainish I am unable to reproduce these two errors even with different proxies, any ideas?

**Comment by ivysrono** [comment]
@gloomy-ghost 这测试，我认输。

**Comment by ivysrono** [comment]
@gloomy-ghost https://travis-ci.org/EFForg/https-everywhere/jobs/203483327#L202
放弃抵抗吧。

**Pull request merged**